manifests/f/Figma/Figma/116.6.2/Figma.Figma.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2023/Feb/03
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Figma.Figma
+PackageVersion: 116.6.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://desktop.figma.com/win/FigmaSetup.exe
+  InstallerSha256: 396C855873A11126099A21410F47161537B03805E4AD92C556DC33076D09980E
+- Architecture: x86
+  InstallerUrl: https://desktop.figma.com/win/FigmaSetup.exe
+  InstallerSha256: 396C855873A11126099A21410F47161537B03805E4AD92C556DC33076D09980E

```

Comment:
```suggestion
  InstallerSha256: 4bce8485d4c1a9c980bd69e0115bc8f23efea8c6fb246f030cfc78df31554123
- Architecture: x86
  InstallerUrl: https://desktop.figma.com/win/FigmaSetup.exe
  InstallerSha256: 4bce8485d4c1a9c980bd69e0115bc8f23efea8c6fb246f030cfc78df31554123
```

Reply:
These match the hashes in https://github.com/microsoft/winget-pkgs/pull/95556.